pkg/monitor/cluster/cluster.go

```diff
@@ -137,18 +137,23 @@ func (mon *Monitor) Monitor(ctx context.Context) (errs []error) {
 		})
 	}
 
-	// If API Server is unreachable, don't need to run the next checks
-	statusCode, err := mon.getAPIServerPingCode(ctx)
-	if err != nil || statusCode != http.StatusOK {
+	// If API is not returning 200, fallback to checking ping and short circuit the rest of the checks
```

Comment:
can we put a comment at the top of this saying: "the apiserver healthz MUST always be emitted first as our APIServerHealthzShort monitor requires it to be emitted always".